lib/src/entity/entity.dart

```diff
@@ -48,10 +48,10 @@ class Entity extends PoolObject<String> {
   }
 
   @override
-  void init([String name]) {
+  void init([String? name]) {
     // id gets set by the entityManager.
     alive = true;
-    this.name = name ?? '';
+    this.name = name;
```

Comment:
It is too bad you can't do like in constructors here

Reply:
Wish we could, but alas to keep the performance up we need the pooling :'(